e2e/resources/enums.ts

```diff
@@ -0,0 +1,43 @@
+// @ts-nocheck
+
+export enum workspaceAccessLevel {
+   OWNER = 'OWNER',
+   READER = 'READER',
+   WRITER = 'WRITER',
+}
+
+export enum workspaceAction {
+   DUPLICATE  = 'Duplicate',
+   DELETE = 'Delete',
+   EDIT = 'Edit',
+   SHARE = 'Share',
+}
+
+const configs = require('../resources/workbench-config');
```

Comment:
Interesting, I did not realize you could just inject a `require` statement like this. Could you add a comment about why this is necessary and what the format of the data expected is?

Reply:
Why aren't you able to just use `import` like in the other files?